web/models/user.ex

```diff
@@ -6,6 +6,7 @@ defmodule Shareguru.User do
     field :email, :string
     field :picture, :string
     field :expire_at, Ecto.DateTime
+    field :hd, :string, virtual: true # Mail domain
```

Comment:
`hd`? why not just `mail_domain`? In fact, we don't even set this value anywhere, from what I can tell. Is it passed from uberauth?